app/util/confluence/upload_attachments.sh

```diff
@@ -4,7 +4,7 @@
 ###################    Variables section         ###################
 # Confluence version variables
 CONFLUENCE_VERSION_FILE="/media/atl/confluence/shared-home/confluence.version"
-SUPPORTED_CONFLUENCE_VERSIONS=(6.13.13 7.0.5 7.4.5)
+SUPPORTED_CONFLUENCE_VERSIONS=(7.4.5 7.0.5 6.13.13)
```

Comment:
do not change the order here, this order is used in by `force` flag to identify the latest version.